### ISSUE TITLE
Remove/move aria-controls in the removable panel prototype

### ISSUE THREAD

**Issue by LJWatson** [bkardell/common-panel#4]
Suggest removing aria-controls from the common-panel-header div in the removable panel example. The header doesn't control the panel content (interacting with
the header doesn't cause a change to the panel content). It also creates a situation where ATs are informed that multiple focusable objects (the header and the remove button) control the panel content, whereas the implied relationship for aria-controls is "one to one" or "one to many", but not "many to one/many".

There may be a case for putting it on the common-panel-remove button instead, since the button does control the panel content to some extent. Would be good to get input from others on this though.

```
<div tabindex="0" class="common-panel-header">
    <i class="common-panel-icon"></i><span>Reminder</span>
    <button aria-controls="-cp-4"  title="Remove this panel" class="common-panel-remove"><i></i></button>
</div>
```

**Comment by bkardell** [comment]
Hmm, I was thinking about the second bit (whether the remove button should have aria-controls) while I was developing it as well....Should it?

If so, in this case you're not actually controlling the content bit though, you're controlling the panel itself (by removing it from the DOM)

**Comment by LJWatson** [comment]
That's why I wasn't sure whether it would be appropriate to apply aria-controls to the button. On the one hand you could argue that it controls the existence of the panel, on the other I'm not sure that falls within the intended use for that attribute.

**Comment by LJWatson** [comment]
Checked in with @stevefaulkner who advises that it wouldn't be appropriate to use aria-controls on a control that dismisses a panel from the UI.

**Comment by bkardell** [comment]
ok good to know, I'll work on the wiring to make sure that aria-controls ISNT on the header if it is neither collapsable nor removable.
